### ISSUE TITLE
Conda installation instructions are incomplete

### ISSUE THREAD

**Issue by izxle** [kivy/kivy#6791]
**Software Versions**
* Python: 3.7.6
* OS: macOS Catalina 10.15.3
* Kivy: 1.11.1
* Kivy installation method: conda

**Describe the bug**
Kivy unable to display window when installed following the  installation instructions using conda.

**Expected behavior**
Kivy should be ready to use after following the installation instructions.

**To Reproduce**
Follow the [conda intallation instructions](https://kivy.org/doc/stable/installation/installation-osx.html#using-conda).

Install Kivy in a new conda environment and run `App().run()`

    conda create --name app
    conda activate app
    conda install kivy -c conda-forge
    python test.py -d

test.py
```python
import kivy
from kivy.app import App
App().run()
```

**Code and Logs and screenshots**
```
[INFO   ] [Logger      ] Record log in /Users/izxle/.kivy/logs/kivy_20-03-22_37.txt
[INFO   ] [Kivy        ] v1.11.1
[INFO   ] [Kivy        ] Installed at "/usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/__init__.py"
[INFO   ] [Python      ] v3.7.6 (default, Jan  8 2020, 13:42:34) 
[Clang 4.0.1 (tags/RELEASE_401/final)]
[INFO   ] [Python      ] Interpreter at "/usr/local/anaconda3/envs/app/bin/python"
[INFO   ] [Factory     ] 184 symbols loaded
[DEBUG  ] [Cache       ] register <kv.lang> with limit=None, timeout=None
[DEBUG  ] [Cache       ] register <kv.image> with limit=None, timeout=60
[DEBUG  ] [Cache       ] register <kv.atlas> with limit=None, timeout=None
[INFO   ] [Image       ] Providers: img_tex, img_imageio, img_dds, img_pil, img_gif (img_pygame, img_ffpyplayer ignored)
[DEBUG  ] [Cache       ] register <kv.texture> with limit=1000, timeout=60
[DEBUG  ] [Cache       ] register <kv.shader> with limit=1000, timeout=3600
[DEBUG  ] [App         ] Loading kv </usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/.kv>
[DEBUG  ] [App         ] kv </usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/.kv> not found
[DEBUG  ] [Window      ] Ignored <pygame> (import error)
[CRITICAL] [Window      ] Unable to find any valuable Window provider. Please enable debug logging (e.g. add -d if running from the command line, or change the log level in the config) and re-run your app to identify potential causes
pygame - ModuleNotFoundError: No module named 'pygame'
  File "/usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/core/__init__.py", line 63, in core_select_lib
    fromlist=[modulename], level=0)
  File "/usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/core/window/window_pygame.py", line 13, in <module>
    import pygame

[CRITICAL] [App         ] Unable to get a Window, abort.
(app) 

```

**Additional context**
It is clear that the cause is that pygame is not installed. In the [Kivy Basics](https://kivy.org/doc/stable/guide/basic.html#installation-of-the-kivy-environment) tutorial it says:

> Kivy depends on many Python libraries, such as pygame, gstreamer, PIL, Cairo, and more. They are not all required, but depending on the platform you’re working on, they can be a pain to install. For Windows and MacOS X, we provide a portable package that you can just unzip and use.
> Have a look at one of these pages for detailed installation instructions:
> - Installation on Windows
> - Installation on OS X
> - Installation on Linux
> - Installation on Raspberry Pi

However, the provided links takes you back to the same incomplete installation instructions. I think pygame should be mentioned in the installation instructions or listed as a prerequisite to use kivy.

Everything works fine after installing pygame, but I get the following warning:

    [WARNING] [Deprecated  ] Pygame has been deprecated and will be removed after 1.11.0: Call to deprecated function __init__ in /usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/core/window/window_pygame.py line 42.Called from /usr/local/anaconda3/envs/app/lib/python3.7/site-packages/kivy/core/__init__.py line 71 by core_select_lib().

**Comment by dbastin** [comment]
Yeah, I just wasted a lot of time on this. Here's hoping I can save someone else some time. 

To install pygame on a Mac OS X, this should work for most situations (including if you use Anaconda)...

`$ sudo pip install -U pip setuptools`
`sudo pip install pygame`

But hopefully this is a blip in the space-time continuum and pygame support will in fact be completely dropped sometime during 1.11.x. 

Note all homebrew options appear to be deprecated, old, and crusty, at least as far as I could tell just now. Not even sure that is a sensible option when using Anaconda. I'm a n00b with all this really. 

To check the pygame install, launch a Python console and... 

`import pygame`

...obvs(?)

✌️

**Comment by matham** [comment]
Pygame is not supported and will be removed at some point. Conda should have installed sdl2 and used that for the window provider - it's not clear to me why that didn't happen.

**Comment by matham** [comment]
Can you please try again installing kivy into a clean new environment and providing the full installation log and running log? I just re-released kivy 1.11.1 on conda so it may already be fixed.
